src/components/layout/Tool.tsx

```diff
@@ -29,21 +31,27 @@ import { Helmet, HelmetProvider } from 'react-helmet-async'
 import { Nullable } from '../../utils/types/types'
 import { AppDispatch, RootState } from '../../redux/store/store'
 import { useLocation } from 'react-router-dom'
+import FullScreenToolModal from '../modal/FullScreenToolModal'
+import { setHandle } from '../../redux/slices/pointsSlice'
+interface ShowModal {
+  active: boolean
+}
 
-const Tool = ({ homepageTrial = false }) => {
+const Tool = ({
+  widthModal = true,
+  defaultHandle = true,
+  homepageTrial = false,
+}) => {
   const dispatch = useDispatch<AppDispatch>()
   const [count, setCount] = useState(3)
-  const [hasArticle, setHasArticle] = useState('')
-  const [hasSomethingTyped, setHasSomethingTyped] = useState('')
   const [latestCopied, setLatestCopied] = useState<{
```

Comment:
why to define object when there's just one variable to store? Why not storing it directly?

Reply:
if our value is fluctuating on synthetic events and I have also performed some js related things that's why I've used react states